modules/web-console/frontend/app/components/ignite-chart/template.pug

```diff
@@ -32,7 +32,7 @@ header
 .ignite-chart-placeholder
     canvas
 
-no-data.no-data
+no-data(result-data-status='$ctrl.resultDataStatus').no-data
```

Comment:
I'm not sure it's a good idea to indirectly couple charts component with AgentManager. Maybe this feature can be opt-in? Like a directive you can add to ignite-chart. Also, it does not support custom statuses, or I'm wrong?

Reply:
I've moved no-data handling to separate component `chart-no-data`.